WordPress/gutenberg
# Demo: Remove core namespace from demo content

Related: https://github.com/WordPress/gutenberg/pull/2950

This pull request seeks to update demo post content to remove the now-redundant `core/` namespace on blocks. Blocks without namespaces are inferred to be the core namespace as of #2950.

__Testing instructions:__

Verify that demo content continues to parse correctly:

1. Navigate to Gutenberg > Demo
2. All blocks should be shown with their respective controls
